validphys2/src/validphys/photon_pdf/compute_photon.py

```diff
@@ -0,0 +1,192 @@
+"""Script that calls fiatlux to add the photon PDF."""
+
+# from validphys import lhapdfset
```

Comment:
`photon` in the name of the file is redundant (`vp.photon_pdf.compute_photon`), since it is already the name of the subpackage.
Taking into account the scope, just limit it to `compute.py`.

Reply:
I would also call the package just `photon`, PDF is the whole scope of `vp`.